test/test_nltk.py

```diff
@@ -0,0 +1,25 @@
+def test_nltk_edit_distance(selenium):
+    selenium.load_package('nltk')
+    selenium.run('import nltk')
+    edit_distance = selenium.run("nltk.edit_distance('foo', 'food')")
+    assert edit_distance == 1
+
+
+def test_nltk_jaccard_distance(selenium):
+    selenium.load_package('nltk')
+    selenium.run('import nltk')
+    jaccard_distance = selenium.run("""
+        nltk.jaccard_distance(set('mapping'), set('mappings'))
+    """)
+    assert jaccard_distance - 0.1428571 < 0.0001
```

Comment:
Better to add an `abs`  (or use `pytest.approx`).

Reply:
Thanks! I went ahead and changed it to use [`pytest.approx`](https://docs.pytest.org/en/latest/reference.html#pytest-approx) (which was new to me, so thanks for pointing it out!).